gitconfig

```diff
@@ -0,0 +1,2 @@
+[remote "origin"]
+    fetch = +refs/pull/*/head:refs/remotes/origin/pr/*
```

Comment:
Isn't this fetching all PRs, regardless of their label?
I'm wondering whether it isn't possible to `git fetch origin ${sha}` or even `git fetch ${shas[@]}`?

Reply:
Good point, I don’t know but I can search